src/goto-instrument/contracts/contracts.h

```diff
@@ -150,7 +149,6 @@ class code_contractst
   goto_functionst &goto_functions;
 
   messaget &log;
-  goto_convertt converter;
```

Comment:
Please remove `#include <ansi-c/goto-conversion/goto_convert_class.h>` from this file and move it to the .cpp file.